runtime/oti/j9consts.h

```diff
@@ -941,6 +941,11 @@ extern "C" {
 #define J9TR_FLAT_RESOLVE_GETFIELD 1
 #define J9TR_FLAT_RESOLVE_PUTFIELD 2
 
+/* Class version minimum for value type support. */
+
+#define VALUE_TYPES_MAJOR_VERSION 67
+#define PREVIEW_MINOR_VERSION 65535
```

Comment:
These constants should be declared in exactly one header file. Since we want the constants in the `J9Consts` namespace for DDR, that place is here. Please remove the duplicate definitions from `j9.h`.